components/project-card/project-card-detailed.jsx

```diff
@@ -91,21 +92,55 @@ class DetailedProjectCard extends React.Component {
     });
   }
 
+  setSocialPanelHeight() {
```

Comment:
I couldn't grab the height of the `<article>` content w/ css so i used js instead to grab the height to calculate the margin offset.

Reply:
Using inline style is bad practice especially for non-animation related stuff.

I'll have to come back to this next week and see if I can come up with other solution. My brain is on Friday afternoon mode now 😂

Reply:
Any thoughts 🤔 ?

Reply:
I know inline styles isn't the most ideal approach, but I think it's the best way to grab these values that I wouldn't be able to do in css. This is similar to the approach we took to building the fellows dir. on foundation-site to keep the page from jumping while the profile cards were loading. 

Can we land as is and create a follow-up ticket if we come up with a better solution to calculate the height offsets?

Reply:
Sure. Can you add some comments in the code explaining why we need this method `setSocialPanelHeight()` and what it does?

Reply:
Sure can.